site/smart-accounts/accounts/light-account.md

```diff
@@ -12,41 +12,49 @@ head:
 
-## Using Light Account
+## Why Light Account
+
+### Gas-optimized (Benchmarks)
+
+Light Account is engineered to minimize gas costs, as demonstrated by the benchmark results below. These results highlight its efficiency in various common user flows:
+
+| Account                                                                                                                 | Native transfer | ERC20 transfer | Creation |
+| ----------------------------------------------------------------------------------------------------------------------- | --------------- | -------------- | -------- |
+| Alchemy LightAccount                                                                                                    | 100844          | 90345          | 279746   |
+| Kernel v2.1-lite                                                                                                        | 101002          | 90321          | 230968   |
+| [SimpleAccount](https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol) | 101319          | 90907          | 383218   |
+| Etherspot                                                                                                               | 103719          | 93324          | 279219   |
+| Biconomy                                                                                                                | 104408          | 93730          | 270013   |
+| Kernel v2.1                                                                                                             | 106460          | 96038          | 265215   |
+| Kernel v2.0                                                                                                             | 110018          | 99622          | 339882   |
 
-The code snippet below demonstrates the usage of Light Account with Account Kit. It creates a Light Account and sends a `UserOperation` from it:
+### Secure (Audited and Open-Sourced)
 
-<!--@include: ../../getting-started.md{56,68}-->
+Light Account has been thoroughly audited by Quantstamp, ensuring its security and reliability. You can find the audit report [here](https://github.com/alchemyplatform/light-account/blob/main/Quantstamp-Audit.pdf). It's an open-source solution, allowing developers to inspect and validate its code.
 
-## Benchmarks
+### Modular 6900 Accounts (Coming Soon)
+
+We've designed Light Account with the future in mind, making it forward-compatible and upgradeable for future, more optimized versions and the upcoming modular [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900) account standard, ensuring you'll stay on the cutting edge of EVM accounts.
```

Comment:
```suggestion
We're pioneering the future of modular smart accounts with [EIP-6900](https://www.alchemy.com/blog/account-abstraction-erc-6900). We're developing a Modular Account to support EIP-6900 coming soon.

Light Account is designed with that future in mind. It is forward-compatible with EIP-6900 so you can upgrade it to 6900-compatible account once the Modular Account is ready.
```

Reply:
also i wasn't sure how to do this, but can you please link the first instance of 'Modular Account' in this comment to the modular account doc in our repo here?